docs/content/latest/releases/whats-new/stable-release.md

```diff
@@ -15,11 +15,63 @@ isTocNested: true
 showAsideToc: true
 ---
 
-{{< note title="Notable features and changes" >}}
-
 Included here are the release notes for all releases in the v2.4 stable release series. Content will be added as new notable features and changes are available in the patch releases of the v2.4 stable release series.
 
-{{< /note >}}
+{{< warning title="Use v2.4.4.0 or later" >}}
+
+YugabyteDB version 2.4.4.0 contains an important fix for a `tcmalloc` memory allocation bug which could lead to process crashes. This was caused by a dependency (the `gperftools` package). If you're using a prior version in the 2.4 series, please upgrade as soon as possible.
```

Comment:
would somehow like to phrase it stronger that it wasn't an actual YugabyteDB code bug, as much as we pulled in a dependency, which itself had this bug...

Reply:
fixed in 514079d